src/pineko/cli/theory_ekos.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+import logging
+
+import click
+import eko
+import rich
+import yaml
+
+from .. import configs, parser, theory_card
+from ._base import command
+
+
+@command.command("theory_ekos")
```

Comment:
I can regroup them - but the scope will be theory (as Emmanuele uses "theory")

Reply:
- things related to theory are now bound below `theory`
- the name `theory` does not relate to the fact, that we're dealing with "theory" stuff here, as opposed to "experimental" - but to the fact that we're expecting a `theory_id` (and hence `theory` is not that bad of a name, although, maybe `TheoryFileAssets` whould be better, but it's too long) and we're iterating over datasets
- now if you say "all we do is like this" - I still want to distinguish between file based operations (`check,compare,convolute,opcard`) and theory based operations; if you really still want to regroup, I can see two options: or the file based move into their own scope and theory becomes root (reverse to the current layout) or we could provide two commands/executables: `pineko` (really meaning `pineko_theory`) and `pineko_file`

Reply:
However, I'm perfectly satisfied with the current layout.

The only difference between this and what I suggested is one name: `theory` in place of `compute`. I still think that they are grouped not because they are related to a theory ID, but because they are part of the theory computation process, instead of being a utility command (so called "file based operation").

> * the name `theory` does not relate to the fact, that we're dealing with "theory" stuff here, as opposed to "experimental" - but to the fact that we're expecting a `theory_id`

That's perfect, so `theory_id` is a wonderful name for a positional argument.